src/sdk/pynni/nni/compression/torch/weight_rank_filter_pruners.py

```diff
@@ -0,0 +1,262 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import logging
+import torch
+from .compressor import Pruner
+
+__all__ = ['L1FilterPruner', 'L2FilterPruner', 'FPGMPruner']
+
+logger = logging.getLogger('torch weight rank filter pruners')
+
+class WeightRankFilterPruner(Pruner):
```

Comment:
`WeightRankFilterPruner` is a little bit complex,  maybe `FilterPruner`?

Reply:
I think `WeightRank` is the emphasis.

Reply:
Ok, that's fine for me.